test/e2e_node/docker_util.go

```diff
@@ -64,51 +54,41 @@ func isDockerLiveRestoreSupported() (bool, error) {
 }
 
 // getDockerInfo returns the Info struct for the running Docker daemon.
-func getDockerInfo() (types.Info, error) {
-	var info types.Info
-	c, err := client.NewClient(defaultDockerEndpoint, "", nil, nil)
+func getDockerInfo(key string) (string, error) {
+	output, err := runCommand("docker", "info", "-f", "{{."+key+"}}")
 	if err != nil {
-		return info, fmt.Errorf("failed to create docker client: %v", err)
+		return "", fmt.Errorf("failed to get docker server version: %v", err)
```

Comment:
Here should be `failed to get docker info` instead.
I guess the line came from line 31 ;-)

Reply:
whoops! fixed.